.github/workflows/mvn-verify-check.yml

```diff
@@ -111,6 +148,16 @@ jobs:
         with:
           distribution: adopt
           java-version: 8
+      
+      - name: Get current date
+        run: |
+          echo "DATE=$(date +'%Y-%m-%d')" >> $GITHUB_ENV
```

Comment:
try implement the key gen suggestion  as a an output of the cache-dependencies job